<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>JetBrainsMono Nerd Font Mono</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Python-Deobfuscation-Multi-Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="filename_label">
       <property name="styleSheet">
        <string notr="true">color: #000;
background-color: #fff;
border: none;
padding: 2px;
font-family: &quot;Arial&quot;;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="textspace_edit">
       <property name="styleSheet">
        <string notr="true">color: #000;
background-color: #fff;
border: 1px solid #000;
font-family: &quot;Arial&quot;;
font-size: 18px;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="selectFile_btn">
         <property name="styleSheet">
          <string notr="true">color: #000;
background-color: #fff;
border: 1px solid #000;
padding: 3px;
font-family: &quot;Arial&quot;;
font-size: 18px;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>Select file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="deobfuscate_btn">
       <property name="styleSheet">
        <string notr="true">color: #000;
background-color: #fff;
border: 1px solid #000;
padding: 3px;
font-family: &quot;Arial&quot;;
font-size: 18px;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Deobfuscate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
